spec/index.bs

```diff
@@ -2379,8 +2428,16 @@ dictionary GPUTextureCopyView {
 };
 </script>
 
+A {{GPUTextureCopyView}} is a view of a sub-region of a [=texture=] [=subresource=] with the initial offset {{GPUOrigin3D} in
+texels, used when copying data between two [=texture=]s, from a [=buffer=] to a [=texture=] and from a [=texture=] to a [=buffer=].
+
+  - For {{GPUTextureDimension/2d}} textures, data is copied between one [=image=] and one [=array layer=].
```

Comment:
this is unclear. One "image" *is* an array layer here

Reply:
I've made some changes here. PTAL, thanks!